doc/manipulating_images/manipulating_images.rst

```diff
@@ -299,6 +310,67 @@ set up your own data preparation procedure:
 * Cleaning signals (e.g., linear detrending, standardization,
   confound removal, low/high pass filtering): :func:`nilearn.signal.clean`
 
+.. _resampling:
+
+Resampling images
+=================
+
+Resampling one image to match another
+-------------------------------------
+
+:func:`nilearn.image.resample_to_img` resamples an image to a reference
+image.
+
+.. topic:: **Example**
+
+    * :ref:`sphx_glr_auto_examples_04_manipulating_images_plot_resample_to_template.py`
+
+.. image:: ../auto_examples/04_manipulating_images/images/sphx_glr_plot_resample_to_template_001.png
+    :target: ../auto_examples/04_manipulating_images/plot_resample_to_template.html
+    :scale: 55%
+
+Resampling specific target affine, shape, or resolution
+--------------------------------------------------------
+
+The resampling procedure takes as input the
+`target_affine` to resample (resize, rotate...) images in order to match
```

Comment:
> somewhat unefficient sentence strucutre

Agreed, but this sentence was already in the doc, I just moved it.
Skipping in the interest of time.